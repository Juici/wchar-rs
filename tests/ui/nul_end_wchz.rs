use wchar::{wchar_t, wchz};

const NUL_END: &[wchar_t] = wchz!("nul byte end\0");

fn main() {}
