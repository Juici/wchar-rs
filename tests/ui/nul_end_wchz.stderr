error: C-style string cannot contain nul characters
 --> $DIR/nul_end_wchz.rs:3:35
  |
3 | const NUL_END: &[wchar_t] = wchz!("nul byte end\0");
  |                                   ^^^^^^^^^^^^^^^^
